akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -53,6 +53,30 @@ final class Flow[-In, +Out, +Mat](private[stream] override val module: Module)
   def via[T, Mat2](flow: Graph[FlowShape[Out, T], Mat2]): Flow[In, T, Mat] = viaMat(flow)(Keep.left)
 
   /**
+   * Transform this [[Flow]] by appending a Reactive Streams compliant [[org.reactivestreams.Processor]]. This
+   * method takes a factory that instantiates the [[org.reactivestreams.Processor]] for every materialization
+   * of the resulting [[Flow]].
+   *
+   * {{{
+   *     +---------------------------------+
+   *     | Resulting Flow                  |
+   *     |                                 |
+   *     |  +------+        +-----------+  |
+   *     |  |      |        |           |  |
+   * In ~~> | this | ~Out~> | processor | ~~> T
+   *     |  |      |        |           |  |
+   *     |  +------+        +-----------+  |
+   *     +---------------------------------+
+   * }}}
+   * The materialized value of the combined [[Flow]] will be the materialized
+   * value of the current flow.
+   */
+  def viaProcessor[T](processorFactory: () ⇒ Processor[Out @uncheckedVariance, T]): Flow[In, T, Mat] = {
```

Comment:
No because "transform" refers to the act of transforming. "via" expresses wiring through a flow-like thing. Therefore, your analogue is wrong. Since "transform" is basically "via a Stage" then it should be called "viaStage" by your logic.